internal/disk/partition_table.go

```diff
@@ -17,9 +17,11 @@ type PartitionTable struct {
 
 	SectorSize   uint64 // Sector size in bytes
 	ExtraPadding uint64 // Extra space at the end of the partition table (sectors)
+
+	RequiredSizes map[string]uint64
```

Comment:
Does this need to be a property of `partitionTable`. I think it doesn't provide any value: `NewPartitionTable` uses it basically as a local variable and `Clone` copies it over, but I don't think there's a code path that would actually use the variable. Please correct me if I'm missing something.

Reply:
Done.